### PR TITLE
chore: cleanup example documentation

### PR DESCRIPTION
**Pull request by jcoeltjen**
- Removed profile flag from docker-compose up/dow command as the compose files do not contain any profile
specific configuration (anymore).

- Changed port for all admin API calls from 9080 to 9180 because the admin api is not exposed on 9080.

- Replaced ip addresses inside service upstream config with compose service names. CIDR for the bridge created by docker-compose might differ between machines, hostnames are fixed.

### CODE REVIEW

File: docs/en/latest/example.md [hunk 1]
```diff
@@ -99,9 +99,7 @@ curl https://web1.lvh.me:9443/ -v --cacert ./mkcert/rootCA.pem
 ### Clean
 
 ```
-docker-compose -p docker-apisix down
+docker-compose down
 
 sudo rm -rf etcd_data/member
-
-rm -rf apisix_log/*.log
```
Comment (tokers) [inline docs/en/latest/example.md:106]: This change is not related to the title. I saw another PR you submitted, it can be change in your another PR.

Comment (jcoeltjen) [reply docs/en/latest/example.md:106]: Fair point. I will update both PRs accordingly.

Comment (jcoeltjen) [reply docs/en/latest/example.md]: I amended the commit, thus the force push.

**Review by tokers — APPROVED** [review]

**Review by soulbird — APPROVED** [review]
